packages/neuron-wallet/script/package.sh

```diff
@@ -0,0 +1,64 @@
+function getplatform() {
+    case $1 in
+        linux|mwl)
+        checkandinstallrpm $1
+        ;;
+        *)
+        package $1
+        ;;
+    esac
+}
+
+function package() {
+    case $1 in 
+        mac)
+        electron-builder --mac
+        ;;
+        win)
+        electron-builder --win --x64
+        ;;
+        linux)
+        electron-builder --linux
+        ;;
+        mwl)
+        electron-builder -mwl
+        ;;
+        *)
+        echo "Please add platform parameters like 'mac' 'win' 'linux' 'mwl' ."
```

Comment:
I suggest by default we build all. That means `mwl` is not required but set as default, so when we run it without any parameter just build all three platforms.